src/SFA.DAS.EmployerIncentives.Api/NLog.local.config

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
+  <extensions>
+    <add assembly="SFA.DAS.NLog.Targets.Redis" />
+    <add assembly="Microsoft.ApplicationInsights.NLogTarget" />
+  </extensions>
+  <variable name="appName" value="das-employer-incentives-api"/>
+  <variable name="simplelayout" value="${longdate} [${uppercase:${level}}] [${logger}] - ${message} ${onexception:${exception:format=tostring}}"/>
+  <targets>
+    <target name="Console" layout="${simplelayout}" xsi:type="Console"/>
+    <target name="Disk" fileName="${currentDir}\logs\${appName}.${shortdate}.log" layout="${simplelayout}" xsi:type="File"/>
```

Comment:
Can you add another file target to just log EI events
e.g.

<target name="EI_Disk" fileName="${currentDir}\logs\${appName}.${shortdate}_EI.log" layout="${simplelayout}" xsi:type="File"/>

and the logger below is

<logger name="SFA.DAS.*" minlevel="Trace" writeTo="EI_Disk" />